src/app/ui/portfolio/PortfolioHeaderBreakdown.tsx

```diff
@@ -63,6 +70,35 @@ export const PortfolioHeaderBreakdown = () => {
     return formattedPositions;
   }, [formattedPositions, portfolioWelcomeScreenClosed]);
 
+  useEffect(() => {
+    if (
+      isLoading ||
+      portfolioHasBeenTracked.current ||
+      !portfolioWelcomeScreenClosed
+    ) {
+      return;
+    }
+    portfolioHasBeenTracked.current = true;
+    const trackingData = parseEarnPortfolioDataToTrackingData(
+      connectedAddresses,
+      tokens,
+      defiPositionGroups,
+    );
+    trackEvent({
+      category: TrackingCategory.Portfolio,
+      action: TrackingAction.PortfolioPageOverview,
+      label: 'portfolio_page_overview',
+      data: trackingData,
+    });
+  }, [
+    trackEvent,
+    portfolioWelcomeScreenClosed,
+    isLoading,
+    tokens,
+    defiPositionGroups,
+    connectedAddresses,
+  ]);
```

Comment:
could you get rid of the dependency list entirely? Since you're using a ref.

```suggestion
  });
```